sim/common/tbc/melee_items.go

```diff
@@ -628,11 +628,12 @@ func init() {
 		const procChance = 2.7 / 60.0
 		actionID := core.ActionID{ItemID: 29996}
 
-		var resourceMetrics *core.ResourceMetrics
+		var resourceMetricsRage *core.ResourceMetrics
+		var resourceMetricsEnergy *core.ResourceMetrics
 		if character.HasRageBar() {
-			resourceMetrics = character.NewRageMetrics(actionID)
+			resourceMetricsRage = character.NewRageMetrics(actionID)
 		} else if character.HasEnergyBar() {
```

Comment:
This shouldn't be an 'else' condition anymore since both rage and energy might be enabled for druids.